lib/checks/text-alternatives/link-text.js

```diff
@@ -0,0 +1,29 @@
+function getText(elem) {
```

Comment:
Could we use `axe.commons.text.accessibleTextVirtual` here? @WilcoFiers, WDYT?

Reply:
@Danidude - Can you also double check the method, I have suggested above to see if it suffices what are you after?

Reply:
@jkodu with the help of the docs you showed me, I saw I could use an other method: `axe.commons.text.accessibleText`